src/main/scala/mesosphere/marathon/core/task/tracker/TaskStatusUpdateProcessor.scala

```diff
@@ -1,9 +1,9 @@
 package mesosphere.marathon.core.task.tracker
 
-import mesosphere.marathon.core.task.bus.TaskStatusObservables
+import org.apache.mesos.Protos.TaskStatus
 
 import scala.concurrent.Future
 
 trait TaskStatusUpdateProcessor {
-  def publish(status: TaskStatusObservables.TaskStatusUpdate): Future[Unit]
+  def publish(status: TaskStatus): Future[Unit]
```

Comment:
Did you think about using the scala case class mesosphere.mesos.protos.TaskStatus instead of the pure proto generated one? Do you see value in that?